descope/tests/mocks/mgmt/managementmock.go

```diff
@@ -278,6 +278,11 @@ type MockUser struct {
 	SearchAllTotalResponse int
 	SearchAllError         error
 
+	SearchAllTestAssert        func(options *descope.UserSearchOptions)
```

Comment:
Change naming in all palces to SearchAllTestUsers

Reply:
changed